metron-platform/metron-writer/src/main/java/org/apache/metron/writer/hdfs/HdfsWriter.java

```diff
@@ -69,14 +66,19 @@ public void init(Map stormConfig, WriterConfiguration configurations) {
 
 
   @Override
-  public void write(String sourceType
+  public BulkWriterResponse write(String sourceType
                    , WriterConfiguration configurations
                    , Iterable<Tuple> tuples
                    , List<JSONObject> messages
                    ) throws Exception
   {
     SourceHandler handler = getSourceHandler(sourceType);
     handler.handle(messages);
+
+    // HDFS SourceHanlder manages writing and rotating files.  Will throw its own Exceptions.
```

Comment:
Will fix typo.  Same deal as previous comment where it's pretty easy to change and manage that directly.